src/backend/graphics/egl.rs

```diff
@@ -497,49 +525,56 @@ impl EGLContext {
             out
```

Comment:
In this block, a `out.push(ffi::egl::NONE as i32);` should be added I believe, so that the attribute list is terminated.

Without this, surface creation fails on my computer with `EGL_BAD_ATTRIBUTE`, but it works fine with it.

(Note that this was already missing prior to this PR.)